Test/Postgres/Interval.hs

```diff
@@ -0,0 +1,23 @@
+{-|
+Module: Test.Postgres.Interval
+Copyright: (c) digitally induced GmbH, 2023
+-}
+module Test.Postgres.Interval where
+
+import Test.Hspec
+import Test.Postgres.Support
+import IHP.Prelude
+import IHP.Postgres.Interval
+import IHP.Postgres.TimeParser
+import Database.PostgreSQL.Simple.ToField
+import qualified Data.Attoparsec.ByteString.Char8 as Attoparsec
+
+tests = do
+    describe "Interval" do
+        describe "Parser" do
+            it "Should Parse" do
+                unpackInterval (PGInterval "25 years 6 mons 4 days 114:00:00.123") `shouldBe`  (PGTimeInterval 25 6 4 410400.123)
```

Comment:
Can we add a test case for uppercase strings like `"25 YEARS 6 MONS 4 DAYS 114:00:00.123"`?

Reply:
The intention of PGInterval should only be wrapping bytestring output from postgres database rather than more arbitrary interval strings from user land. Probably want to keep this as fast possible given the strings will have a definite formatted style?

Reply:
ok makes sense 👍